apps/nuxt3-ssr/interfaces/types.ts

```diff
@@ -254,6 +254,7 @@ export interface IColumn {
 }
 
 export interface ITableMetaData {
+  name: string;
```

Comment:
do you still want to use that for your application? I would expect you want to use only 'id' and 'label'
(nb. label defaults to 'name' unless seperate labels are defined)

Reply:
done